app/javascript/gobierto_data/webapp/components/commons/RecentQueries.vue

```diff
@@ -38,7 +38,10 @@ export default {
   created() {
     document.addEventListener("keyup", this.onKeyUpNextRecentItem)
   },
-  beforeDestroy() {
+  activated() {
+    document.addEventListener("keyup", this.onKeyUpNextRecentItem)
+  },
+  deactivated() {
```

Comment:
This is what I say about the `keep-alive`. Why `RecentQueries` should be keep alive? it's only present at DataTab, if I remember